src/MSBuild.UnitTests/TerminalLogger_Tests.cs

```diff
@@ -449,6 +450,22 @@ public Task PrintBuildSummary_2Projects_FailedWithErrorsAndWarnings()
             return Verify(_outputWriter.ToString(), _settings).UniqueForOSPlatform();
         }
 
+        [Fact]
+        public Task PrintProjectOutputDirectoryLink()
+        {
+            // Send message in order to set project output path
+            BuildMessageEventArgs e = MakeMessageEventArgs(
+                    $"本地化 -> {_projectFileWithForeignSymbols.Replace("cproj", "dll")}",
```

Comment:
It doesn't contain `cproj` now, only `.proj`--intentional?

Reply:
Yes, in order to reflect real scenario. But it really can be anything, won't affect the test

Reply:
What I'm saying is that this `Replace()` call doesn't do anything. Did you want it to?

Reply:
Oh, sorry read the comment wrong.
It is a typo, should be proj as well. Thanks!